MBBSEmu/Session/SessionBase.cs

```diff
@@ -205,7 +210,11 @@ public string Username
         /// <param name="dataToSend"></param>
         public bool SendToClientAsync(byte[] dataToSend)
         {
-            return DataToClient.TryAdd(dataToSend.Where(c => shouldSendToClient(c)).ToArray());
+            if (OutputEnabled)
```

Comment:
This needs to be added to SendToClient() as well

Reply:
Good call